elastic/kibana
# [Lens] Fixes color mappings special tockens i18n

## Summary

fix https://github.com/elastic/kibana/issues/168151

To test:
- create a cartesian charts with breakdowns that contains both the `Other` group and an `(empty)` value
- apply a manual color mapping that use both these terms by select them from the combo-box,  not by typing them.
- Then change the Kibana language to a different one other than English and see if the color mapping applies correctly.
